### ISSUE TITLE
[SensioLabsInsight] exit() and die() functions should be avoided

### ISSUE THREAD

**Issue by ricknox** [ricknox/Butler#3]
### Issue overview card

| Q                | A
| ---------------- | -----
| Bug report?      | yes
| Feature request? | no
| BC Break report? | no
| RFC?             | no
| Butler version   | 0.0.1

_Posted from [SensioLabsInsight](https://insight.sensiolabs.com/projects/88f55f67-dd1c-473d-a8c6-1d7c71e3509b/analyses/1)_

### Description

[in src/Command/SelfUpdateCommand.php, line 58](https://github.com/ricknox/Butler/blob/8add3b57e94ab02d7bb1e533f170c01449af5659/src/Command/SelfUpdateCommand.php#L58) the line stops the execution flow, without explanation. If this is for debug, you should remove it. If this is to deal with an error, use exceptions instead.

```php
            } else {
                echo "You are already using the latest version available.\n";
            }
        } catch (\Exception $e) {
            echo "Something went horribly wrong. Check your internet connection and try again.\n";
            exit(1);
        }
    }
}

```

**Comment by ricknox** [comment]
Created a pull request #4 to fix this issue.

**Comment by ricknox** [comment]
Merged in e0fd01f53553496ba4c253e3ad51d89bcd70f872.
